mlkem/native/x86_64/src/arith_native_x86_64.h

```diff
@@ -5,17 +5,16 @@
 #ifndef MLKEM_X86_64_NATIVE_H
 #define MLKEM_X86_64_NATIVE_H
 
-#include "common.h"
+#include "../../../common.h"
 
 #include <immintrin.h>
 #include <stdint.h>
+#include "../../../polyvec.h"
 #include "consts.h"
-#include "fips202.h"
-#include "polyvec.h"
 
-#define REJ_UNIFORM_AVX_NBLOCKS \
-  ((12 * MLKEM_N / 8 * (1 << 12) / MLKEM_Q + SHAKE128_RATE) / SHAKE128_RATE)
-#define REJ_UNIFORM_AVX_BUFLEN (REJ_UNIFORM_AVX_NBLOCKS * SHAKE128_RATE)
+#define REJ_UNIFORM_AVX_NBLOCKS 3 /* See MLKEM_GEN_MATRIX_NBLOCKS */
+#define REJ_UNIFORM_AVX_BUFLEN \
+  (3 * 168) /* REJ_UNIFORM_AVX_BUFLEN * SHAKE128_RATE */
```

Comment:
I don't see how this change is related to this commit. Please do it in a separate commit.

Reply:
Fair point. This is to avoid reliance on the FIPS202 header.

Reply:
Ahh okay - the `SHAKE128_RATE` - I see.